assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -284,7 +284,7 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
         set(frame.rotate);
       }}
       onRotate={({ target, beforeRotate }) => {
```

Comment:
There's also the multi-selection movable for handling multi-selection rotation, we should fix it there, too.

Reply:
Done.